### PR TITLE
Build: the completion of specified cc_library

### PR DESCRIPTION
**Pull request by quas-modo**
### What problem does this PR solve?

Issue Number: [#2322](https://github.com/opencurve/curve/issues/2322)

Problem Summary: The current [Makefile](https://github.com/opencurve/curve/blob/master/Makefile) of Curve only supports the compilation of cc_binary.

### What is changed and how it works?

What's Changed: /utils/build-in-image.sh has been changed

How it Works: If anyone run `make build/ make ci-build` and have the parameter only to sepecify the target as cc_library, then the change will support to compiler the cc_library.n Also, when running `make list`, the change will list cc_library target.

the success completion of the issue's examples

Side effects(Breaking backward compatibility? Performance regression?): None

### Check List

- [] Relevant documentation/comments is changed or added
- [] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by quas-modo** [comment]
cicheck

**Comment by quas-modo** [comment]
cicheck

File: thirdparties/etcdclient/Makefile [hunk 1]
```diff
@@ -18,9 +18,9 @@ SHELL=/bin/bash
 
 pwd := $(shell pwd)
 
-all: intall-go install-etcdclient libetcdclient
+all: install-go install-etcdclient libetcdclient
 
-intall-go:
+install-go:
```
Comment (Wine93) [inline thirdparties/etcdclient/Makefile:23]: :)

File: util/basic.sh [hunk 2]
```diff
@@ -13,3 +13,8 @@ die() {
     msg "\33[31m[✘]\33[0m ${1}${2}"
     exit 1
 }
+
+print_title() {
+    local delimiter=$(printf '=%.0s' {1..20})
+    msg "$delimiter [$1] $delimiter\n"
+}
```
Comment (Wine93) [inline util/basic.sh:20]: Add a newline here.

Comment (quas-modo) [reply util/basic.sh]: Done

**Comment by quas-modo** [comment]
cicheck

**Comment by quas-modo** [comment]
If there's anything that needs to be changed, I'm more than happy to keep making changes.
Also, maybe I can do some annotations and documentation changes as well.

**Comment by quas-modo** [comment]
cicheck

**Comment by Wine93** [comment]
cc @Cyber-SiKu

**Comment by Cyber-SiKu** [comment]
@quas-modo pls 如resolved conflicts

**Comment by quas-modo** [comment]
cicheck

**Comment by quas-modo** [comment]
cicheck

**Comment by quas-modo** [comment]
cicheck

File: util/playground.sh [hunk 3]
```diff
@@ -8,7 +8,7 @@
 g_obm_cfg=".obm.cfg"
 g_worker_dir="/curve"
 g_container_name="curve-build-playground.master"
-g_container_image="opencurvedocker/curve-base:build-debian9"
+g_container_image="opencurvedocker/curve-base:build-debian11"
```
[the remark this answers is not in the document]
Comment (quas-modo) [reply util/playground.sh:11]: I am trying to fix it.

**Comment by quas-modo** [comment]
cicheck

**Comment by quas-modo** [comment]
cicheck

**Comment by quas-modo** [comment]
cicheck

**Comment by quas-modo** [comment]
cicheck

**Comment by quas-modo** [comment]
cicheck

**Comment by quas-modo** [comment]
cicheck

**Comment by quas-modo** [comment]
cicheck

**Comment by quas-modo** [comment]
cicheck

**Comment by quas-modo** [comment]
cicheck

**Comment by quas-modo** [comment]
cicheck

**Comment by quas-modo** [comment]
cicheck

**Comment by quas-modo** [comment]
cicheck

**Comment by quas-modo** [comment]
cicheck

**Comment by quas-modo** [comment]
cicheck

**Comment by quas-modo** [comment]
cicheck

**Review by Wine93 — APPROVED** [review]
LGTM!

**Review by Cyber-SiKu — APPROVED** [review]

**Pull request closed without merging**